lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -994,16 +1002,18 @@ def _mean_and_variance_to_probabilities(self, mean_values, variance_values,
         probability_cube = probability_cube_template.copy(data=probabilities)
         return probability_cube
 
-    def process(self, mean_values, variance_values, probability_cube_template):
+    def process(self, location_parameter_values, scale_parameter_values,
+                probability_cube_template):
         """
-        Generate probabilities from the mean and variance of distribution.
+        Generate probabilities from the location and scale parameters of the
+        distribution.
 
         Args:
-            mean_values (iris.cube.Cube):
-                Cube containing the distribution mean values of a diagnostic,
-                e.g. the mean over realizations.
-            variance_values (iris.cube.Cube):
-                Cube containing the distribution variance values of a
+            location_parameter_values (iris.cube.Cube):
+                Cube containing the distribution location parameter values of
+                a diagnostic, e.g. the mean over realizations.
```

Comment:
I'm not sure the e.g. actually helps here

Reply:
I've removed this.